src/module_wrap.cc

```diff
@@ -622,6 +628,50 @@ void ModuleWrap::SetImportModuleDynamicallyCallback(
   iso->SetHostImportModuleDynamicallyCallback(ImportModuleDynamically);
 }
 
+void ModuleWrap::InitializeImportMetaObject(Local<Context> context,
+                                            Local<Object> meta) {
+  Isolate* isolate = context->GetIsolate();
+  Environment* env = Environment::GetCurrent(isolate);
+  meta->CreateDataProperty(context, env->url_string(),
+                           PersistentToLocal(isolate, this->url_)).FromJust();
+}
+
+void ModuleWrap::HostInitializeImportMetaObjectCallback(
+    Local<Context> context, Local<Module> module, Local<Object> meta) {
+  Isolate* isolate = context->GetIsolate();
+  Environment* env = Environment::GetCurrent(isolate);
```

Comment:
Tiny nit: Can you use `Environment::GetCurrent(context)`? Getting it from the isolate only looks up the context on the isolate and takes it from there

Reply:
Ah thanks! I didn't know there was a `Environment::GetCurrent(context)`